.github/actions/setup-rust/action.yml

```diff
@@ -25,11 +25,13 @@ runs:
       if: inputs.os == 'windows'
       shell: pwsh
       run: |
-        if( (-not (get-command rustup))) {
-          # TODO: Install Rust on Windows
-          throw "Rustup not found"
+        if (-not (Get-Command rustup -ErrorAction SilentlyContinue)) {
+          Write-Host "Installing Rust..."
```

Comment:
Alternatively review if the following action can be used / work
https://github.com/actions-rust-lang/setup-rust-toolchain